packages/0x.js/src/contract_wrappers/generated/.gitignore

```diff
@@ -0,0 +1,6 @@
+dummy_token.ts
+ether_token.ts
+exchange.ts
+token_registry.ts
+token_transfer_proxy.ts
+token.ts
```

Comment:
Why not add these to the top-level gitignores with the full path?

Reply:
Because I think it's nicer to have them here?
If the user is already here he/she has more context and is more probably understanding why are those here.

Reply:
ok